scss/_carousel.scss

```diff
@@ -182,37 +182,20 @@
   list-style: none;
 
   li {
-    position: relative;
     flex: 0 1 auto;
     width: $carousel-indicator-width;
-    height: $carousel-indicator-height;
+    height: ($carousel-indicator-height + 20px);
     margin-right: $carousel-indicator-spacer;
     margin-left: $carousel-indicator-spacer;
     text-indent: -999px;
     cursor: pointer;
     background-color: $carousel-indicator-active-bg;
+    background-clip: padding-box;
+    // Use transparent borders to increase the hit area by 10px on top and bottom.
+    border-top: 10px solid transparent;
```

Comment:
Can we move to a variable for the `10px` value, and then use that here, on L196 and L187?

Reply:
Changed it to a variable and used `box-sizing: content-box` instead of adding the hit area height to indicator height (this makes it possible to define the hit area height in another unit).